sql/moz-fx-data-shared-prod/search_terms_derived/search_terms_daily_v1/query.sql

```diff
@@ -0,0 +1,31 @@
+WITH query_logs AS (
+  SELECT
+    DATE(timestamp) AS merino_date,
+    jsonPayload.fields.*
+  FROM
+    `suggest-searches-prod-a30f.logs.stdout`
```

Comment:
Unsanitized search terms tables are not accessible to standard bqetl machinery: see the comment in https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v2_external/query.sql#L3.

We currently have two production mechanisms for accessing unsanitized search terms data:
1. BQ scheduled query within the unsanitized project, as was done with `suggest_impression_sanitized_v2`
https://github.com/mozilla-services/cloudops-infra/blob/master/projects/merino/tf/modules/log-routing/sanitizer.sql
https://github.com/mozilla-services/cloudops-infra/blob/master/projects/merino/tf/modules/log-routing/main.tf#L187

2. via a custom k8s job https://github.com/mozilla/search-terms-sanitization

@chelseatroy may have an opinion on how to proceed here. The caveat for the scheduled query approach is that it requires a bqetl dag to copy the data from the merino project (where the scheduled query is contrained to write to) into shared-prod.

Reply:
I _think_ the easiest thing might end up being a virtual view, really. We did this with the languages table: Some SQL runs on a regular cadence and populates that table right from bigquery-etl. That can be found in `bigquery-etl/sql/moz-fx-data-shared-prod/search_terms`. It consists of some metadata describing the required access and then a SQL query and that's it.

Reply:
My understanding was that we were creating an aggregate of **sanitized** search terms.

Reply:
If we're creating an aggregate of sanitized search terms then perhaps all that's needed here is to change the source dataset to that sanitized location and obviate the cross-project machinery required to access the unsanitized search terms.

Reply:
Yes - we're just aggregating sanitized search terms. Sorry about the confusion! I fixed the SQL.